src/C_API.jl

```diff
@@ -111,6 +111,8 @@ mutable struct InterfaceData
     lb::Vector{Cdouble}
     ub::Vector{Cdouble}
     z::Vector{Cdouble}
+    variable_name::Vector{String}
+    constraint_name::Vector{String}
```

Comment:
A minor thing, but probably `variable_names` and `constraint_names`, since these are plural.